### PR TITLE
[ci] use background script

### PR DESCRIPTION
**Pull request by fkorotkov**
There is a special instruction in Cirrus for that https://cirrus-ci.org/guide/writing-tasks/#background-script-instruction

Might be related to https://github.com/flutter/flutter/issues/90078

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Note that unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I [updated pubspec.yaml](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code

### CODE REVIEW

**Review by ditman — APPROVED** [review]
LGTM! Let's try this and see if it helps the repo recover!

File: .cirrus.yml [hunk 1]
```diff
@@ -208,12 +208,12 @@ task:
...
           CHANNEL: "master"
           CHANNEL: "stable"
-      install_script:
+      install_background_script:
         - git clone https://github.com/flutter/web_installers.git
         - cd web_installers/packages/web_drivers/
         - dart pub get
         - dart lib/web_driver_installer.dart chromedriver --install-only
-        - ./chromedriver/chromedriver --port=4444 &
+        - ./chromedriver/chromedriver --port=4444
```
Comment (stuartmorgan) [inline .cirrus.yml:216]: We probably want this to be a separate script step that is background, and the rest to stay as-is.

Comment (ditman) [reply .cirrus.yml:216]:
```
      install_script:
        - git clone https://github.com/flutter/web_installers.git
        - cd web_installers/packages/web_drivers/
        - dart pub get
        - dart lib/web_driver_installer.dart chromedriver --install-only
      install_background_script:
        - ./chromedriver/chromedriver --port=4444
```

?

Comment (fkorotkov) [reply .cirrus.yml:217]: See 02dac56e0d663693c813aee7e11c7ae66791371c

**Review by ditman — COMMENTED** [review]
This is probably going to fail:

```
dart lib/web_driver_installer.dart chromedriver --install-only
Error when reading 'lib/web_driver_installer.dart': No such file or directory.
```

Does this need to `cd` at the beginning of the `install_background_script` (similarly to how we `cd web_installers/packages/web_drivers/` at the beginning of `install_script`?)

**Comment by ditman** [comment]
I'll merge on red when the rest of the tests pass. Thanks @fkorotkov for the super fast fix!

**Comment by ditman** [comment]
@fkorotkov is this missing a log file? https://cirrus-ci.com/task/5474720939245568?logs=chromedriver

**Comment by fkorotkov** [comment]
@ditman it might failed to stream in real-time and will try to recover once the whole tasks finishes... Let's wait.

**Comment by stuartmorgan** [comment]
Do we know why the old version suddenly broke?

**Comment by ditman** [comment]
Web tests are running again:

* https://cirrus-ci.com/task/5474720939245568?logs=drive#L104
* https://cirrus-ci.com/task/6037670892666880?logs=drive#L57

**Comment by fkorotkov** [comment]
@stuartmorgan yeah, seems this is the reason https://github.com/flutter/flutter/issues/90078#issuecomment-919440658

**Comment by stuartmorgan** [comment]
> I'll merge on red when the rest of the tests pass. Thanks @fkorotkov for the super fast fix!

Since this can't possible affect any tasks other than the one in changed, and those are green, I'm going to land now to green the tree.

**Pull request merged**